components/Tabs/README.md

```diff
@@ -2,72 +2,98 @@
 title: "Tabs"
 layout: detail
 section: components
-excerpt: "Tabs make it easy to explore and switch between different views."
+excerpt: "Material Design tabs are bars of buttons used to navigate between groups of content."
 iconId: tabs
 path: /catalog/tabs/
+api_doc_root: true
 -->
 
-# Tabs
+<!-- This file was auto-generated using ./scripts/generate_readme Tabs -->
 
-<div class="article__asset article__asset--screenshot">
-  <img src="docs/assets/tabs.png" alt="Tabs" width="375">
-</div>
+# Tabs
 
 Tabs are bars of buttons used to navigate between groups of content.
 
-## Design & API Documentation
+<img src="docs/assets/tabs.png" alt="Tabs" width="375">
+
```

Comment:
Need to keep the screenshot surrounding tags.

Reply:
done.

Reply:
done.